package.json

```diff
@@ -181,6 +184,8 @@
     "@babel/eslint-parser": "^7.19.1",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-proposal-private-methods": "^7.18.6",
```

Comment:
Some `-proposal-` plugins were renamed in `stripes-webpack` to `-transform-` because they became standard and will were renamed by `babel`.
https://github.com/folio-org/stripes-webpack/pull/114
Please rename the same plugins here and in `babel.config.js`

Reply:
@BogdanDenis done. Thanks for information